docs/code-of-conduct.md

```diff
@@ -1,97 +1,105 @@
-  sending private messages to individuals, so that the first available
-  expert can answer, and the answers are available to the whole
-  community.
-
-Examples of unacceptable behavior by participants include:
-
-* The use of sexualized language or imagery and unwelcome sexual attention or
-  advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
-* Criticism of contributions which is not constructive, which occurs outside
-  project channels or which is repetitive
-* Publishing others' private information, such as a physical or electronic
-  address without explicit permission
-
-We will exclude you from interaction if you insult, demean or harass anyone.
-That is not welcome behavior.
-
-## Responsibilities of Project Maintainers
-
-Project maintainers are responsible for clarifying the standards of acceptable
-behavior and are expected to take appropriate and fair corrective action in
-response to any instances of unacceptable behavior.
-
-Project maintainers have the right and responsibility to remove, edit, or reject
-comments, commits, code, wiki edits, issues, and other contributions that are
-not aligned to this Code of Conduct, or to ban temporarily or permanently any
-contributor for other behaviors that they deem inappropriate, threatening,
-offensive, or harmful.
+Like the technical community as a whole, the Zulip team and community is
+made up of a mixture of professionals and volunteers from all over the
+world, working on every aspect of the mission, including mentorship,
+teaching, and connecting people.
+
+Diversity is one of our huge strengths, but it can also lead to
+communication issues and unhappiness. To that end, we have a few ground
+rules that we ask people to adhere to. This code applies equally to
+founders, mentors and those seeking help and guidance.
+
+This isn't an exhaustive list of things that you can't do. Rather, take it
+in the spirit in which it's intended --- a guide to make it easier to enrich
+all of us and the technical communities in which we participate.
+
+## Expected Behavior
+
+The following behaviors are expected and requested of all community members:
+
+* Participate. In doing so, you contribute to the health and longevity of
+  the community.
+* Exercise consideration and respect in your speech and actions.
+* Attempt collaboration before conflict. Assume good faith.
+* Refrain from demeaning, discriminatory, or harassing behavior and speech.
+* Take action or alert community leaders if you notice a dangerous
+  situation, someone in distress, or violations of this code, even if they
+  seem inconsequential.
+* Community event venues may be shared with members of the public; be
+  respectful to all patrons of these locations.
+
+## Unacceptable Behavior
+
+The following behaviors are considered harassment and are unacceptable
+within the Zulip community:
+
+* Jokes or derogatory language that singles out members of any race,
```

Comment:
It's technically correct, I think (my impression is in "A or B VERB", verb is conjugated according to B), but I recognize that it sounds awkward. 

I do want to put Jokes first, since I expect that's the majority of CoC violations. But if you have suggestion for a better phrasing lmk.